packages/react/package.json

```diff
@@ -0,0 +1,39 @@
+{
```

Comment:
Not sure where to add this but should we add a propper README to this one straight away?